cmd/pgngen/deduper.go

```diff
@@ -0,0 +1,30 @@
+package main
```

Comment:
@george-e-shaw-iv what's the right way to do this?  make this an internal package -- internal/deduper/deduper.go, package deduper, and just pull it in from pgngen?  or an internal package only used for one of the cmd tools, is it right to just put it in the cmd/xxx/ dir?

Reply:
defer for next PR.